README.md

```diff
@@ -1,3 +1,10 @@
+<br/>
+<a>
```

Comment:
Did you introduce the anchor element to link to micro.ros.org? Or shall it serve as anchor for navigation in the page?

Reply:
Removed here 8e8be294e31b1cfa16adf3f710995965b7556a77